src/api/config/routes/api.rb

```diff
@@ -181,6 +181,11 @@
     get 'public/about' => 'about#index'
     get 'public/configuration' => :configuration_show
     get 'public/configuration.xml' => :configuration_show
+    # FIXME: We have no clients anymore that use this
+    get 'lastevents' => :lastevents
```

Comment:
We could deprecate this route and create a redirection to the final endpoint we want to leave, before retiring the endpoint.

There are some other redirections for this same purpose. For example: https://github.com/openSUSE/open-build-service/blob/3eba67d31bceddfbd61f89072ff4691e7d9c1953/src/api/config/routes/webui.rb#L82-L86

What do you think?

Reply:
yeah I'm going to discuss this with the backend folks...